staging/prometheus-operator/dependent-charts/kube-state-metrics/README.md

```diff
@@ -15,7 +15,7 @@ $ helm install stable/kube-state-metrics
 | Parameter                               | Description                                                                           | Default                                    |
 |:----------------------------------------|:--------------------------------------------------------------------------------------|:-------------------------------------------|
 | `image.repository`                      | The image repository to pull from                                                     | quay.io/coreos/kube-state-metrics          |
-| `image.tag`                             | The image tag to pull from                                                            | `v1.6.0`                                   |
```

Comment:
Are there no other changes from the upstream chart?

Reply:
Nope, https://github.com/helm/charts/commit/3c88bd086d8d04d29f8dbf5b8b038f65b082949b

Reply:
This is changing from `v1.6.0` https://github.com/helm/charts/blame/0e5236eea2bc1a9dc7904f25cda80983acbc158e/stable/kube-state-metrics/Chart.yaml to master.

Doesn't look like I can filter for files but https://github.com/helm/charts/compare/0e5236eea2bc1a9dc7904f25cda80983acbc158e...master shows 12 files changed under `kube-state-mterics/`.

Reply:
those are 2k file changes... :D

Reply:
I did copy over all the changes which changed the labels 🤞